### PR TITLE
Change ios-app:GetUrl target to ios-app

### PR DESCRIPTION
**Pull request by c-parsons**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -67,10 +67,10 @@ response in a text view.
 You can build the application by running:
 
 ```
-$ bazel build //ios-app:GetUrl
+$ bazel build ios-app --ios_sdk_version=8.4
```
Comment (kchodorow) [inline README.md]: Why this change? Do users need to change this to the version they want to build with?

Comment (c-parsons) [inline README.md]: honestly, I'm not really sure why this worked before, perhaps default behavior changed? I was getting complaints about UiKit not being supported with my current default sdk version, so this takes out the guesswork.

If this is a not-ideal step, I can delve deeper and figure out how to have the app supported with the default sdk version

Alternatively, do you want me to explain this flag in the readme?

Comment (kchodorow) [inline README.md]: We should know 1) will this break people with older/newer versions of XCode? 2) Does this need to be set to something different for other versions of xcode?

Minor thing: just to be consistent with targets above, please make it //ios-app.

Comment (c-parsons) [inline README.md]: I'm not entirely sure - it is possible I indeed updated my xcode since my earlier revision. iOS SDK 8.4 is supported in XCode 6.4, which at the time of this writing seems to be apple's current stable version. I've updated the readme to somewhat explain this fact, is this sufficient?

Comment (kchodorow) [inline README.md]: That looks good, thanks!
